packages/wiki-docs/Contributing.md

```diff
@@ -203,4 +203,104 @@ npx @seasketch/geoprocessing@0.15.1-beta.1 init 0.15.1-beta.1
 
 # Wiki
 
-Diagrams are maintained in internal SeaSketch [drive share](https://drive.google.com/drive/folders/1JL7BkOf2mP2VaXQKlM2kkENqHW9LtCbm?usp=sharing)
\ No newline at end of file
+Diagrams are maintained in internal SeaSketch [drive share](https://drive.google.com/drive/folders/1JL7BkOf2mP2VaXQKlM2kkENqHW9LtCbm?usp=sharing)
+
+# i18n
+
+Localization is managed using [i18next](https://react.i18next.com/). Wrap _all_ strings displayed in report client UIs in appropriate tags so they can be translated into multiple languages. ESLint rules will flag missing tags.
```

Comment:
eslint rules should probably be optional for report implementations outside the monorepo. Could get quite annoying otherwise.